<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulatorPreferences</class>
 <widget class="QWidget" name="SimulatorPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>236</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="randomGroup">
     <property name="title">
      <string>&amp;Random run</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="2" column="0">
       <widget class="QLabel" name="stepsLabel">
        <property name="text">
         <string>&amp;Number of steps:</string>
        </property>
        <property name="buddy">
         <cstring>stepsSpin</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="stepsSpin"/>
      </item>
      <item row="7" column="0">
       <widget class="QCheckBox" name="randomBox">
        <property name="text">
         <string>R&amp;andom seed</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="seedLabel">
        <property name="text">
         <string>&amp;Seed:</string>
        </property>
        <property name="buddy">
         <cstring>seedSpin</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="seedSpin">
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="delayLabel">
        <property name="text">
         <string>&amp;Delay [ms]:</string>
        </property>
        <property name="buddy">
         <cstring>delaySpin</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="delaySpin">
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="divineBox">
     <property name="title">
      <string>Verification</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QSpinBox" name="threadsSpin">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="threadsLabel">
        <property name="text">
         <string>Number of &amp;workers:</string>
        </property>
        <property name="buddy">
         <cstring>threadsSpin</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="algorithmLabel">
        <property name="text">
         <string>Default algorithm:</string>
        </property>
        <property name="buddy">
         <cstring>algorithmBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="algorithmBox">
        <item>
         <property name="text">
          <string>Nested DFS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OWCTY</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MAP</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>stepsSpin</tabstop>
  <tabstop>delaySpin</tabstop>
  <tabstop>seedSpin</tabstop>
  <tabstop>randomBox</tabstop>
  <tabstop>algorithmBox</tabstop>
  <tabstop>threadsSpin</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
